requirements.txt

```diff
@@ -0,0 +1,7 @@
+dronekit==2.1.0
+dronekit-sitl==3.0.1
+nose==1.3.7
+psutil==3.4.2
+pymavlink==1.1.70
+requests==2.9.1
```

Comment:
i didn't explicitly install requests might be a dep from something else, ill remove it